website/client/src/components/userMenu/profile.vue

```diff
@@ -836,8 +841,23 @@ export default {
       const profileUserId = this.userId;
 
       if (profileUserId && profileUserId !== this.userLoggedIn._id) {
-        const response = await this.$store.dispatch('members:fetchMember', { memberId: profileUserId });
-        user = response.data.data;
+        try {
+          const response = await this.$store.dispatch('members:fetchMember', {
+            memberId: profileUserId,
+          });
+          // debugger;
+          user = response.data.data;
+        } catch (e) {
+          // user = null;
+          user.active = false;
+          console.log('fkdsafj;dsak;', user.active);
```

Comment:
this should be removed, as well as the other comments like // debugger, //user = null